packages/sign_in_with_apple/sign_in_with_apple/android/build.gradle

```diff
@@ -2,7 +2,7 @@ group 'com.aboutyou.dart_packages.sign_in_with_apple'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.31'
```

Comment:
How come this now points to 1.5, when initially it was supposed to be 1.6?

Reply:
Initially for 2.10 only `1.5.31` was the minimum required version as per https://docs.flutter.dev/release/breaking-changes/kotlin-version